### DISCUSSION TITLE
Support for gpoddersync

### DISCUSSION THREAD

**Discussion by eyck-dev** [gilcu3/hullcaster#8] [Ideas]
Does the current version support gpoddersync(https://apps.nextcloud.com/apps/gpoddersync)? This is pretty solid for syncing and Kasts also supports it. It feels way more reliable than gpodder .net

**Comment by gilcu3** [comment]
Hi, currently I am using `hullcaster` with the gpodder api endpoints, but only tested with [opodsync](https://github.com/kd2org/opodsync). 

I was using `Kasts` previously too, and if I remember correctly it had some issue with the [gpodder api](https://gpoddernet.readthedocs.io/en/latest/api/index.html), but not with the nextcloud `gpoddersync`. Anyway, I think I can try and support these endpoints as well, given its similarity with the original. 

Feel free to make a PR on this if you get to do it before I can. My end goal is to support the [open podcast api](https://openpodcastapi.org/) once it comes out, which should fix issues like syncing the queue.

**Reply by eyck-dev** [reply]
I'm not too familiar with programming but I will try to look at the code. Thanks a lot for the response.
